modules/stb_vorbis/doc_classes/AudioStreamOGGVorbis.xml

```diff
@@ -13,6 +13,29 @@
 	<members>
 		<member name="data" type="PackedByteArray" setter="set_data" getter="get_data" default="PackedByteArray(  )">
 			Contains the audio data in bytes.
+			You can load a file without import using the method below. Keep in mind that this method loads the whole file into memory and may be not the prefered way for big files.
```

Comment:
```suggestion
			You can load a file without import using the method below. Keep in mind that this method loads the whole file into memory and may not be ideal for huge files (several hundred MBs or more).
```

Reply:
This method is actually not longer possible with the change to libvorbis in #52406. I am going to remove all of this